webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Parameters/ViewTableRow.js

```diff
@@ -11,11 +11,13 @@ const getValue = param => {
   if (param['hidden_value?']) {
     return '••••••••';
   }
+  if (!param.value && param.value !== false)
```

Comment:
```suggestion
  if (param.value === '')
```

Reply:
`''` values are serialised to null , so I updated it as `if (param.value === null)`